### ISSUE TITLE
Single object importing (NSManagedObject+RZImport)

### ISSUE THREAD

**Issue by bsmithraiz** [Raizlabs/RZVinyl#51]
If I import an array of objects using rzi_objectsFromArray: inContext:, it will do an existing object check automatically and not create duplicates. But if I import a single object using rzi_objectFromDictionary: inContext:, it will not do an existing object check and will create duplicates. It does check to see if rzi_existingObjectForDict: is defined, but I still think it is confusing that the array call checks and the dictionary call doesn't. I'd suggest that if rzi_existingObjectForDict: is not defined, it makes a duplicate check automatically (which seems like what 95% of defined rzi_existingObjectForDict: methods would do anyway).

TL:DR 
rzi_objectsFromArray: inContext: checks for duplicates and rzi_objectFromDictionary: inContext: doesn't

**Comment by KingOfBrian** [comment]
Are you still seeing this @bsmith11 ? The code appears to be properly checking for existing objects on the singular version, but I'm probably missing something.
